### DISCUSSION TITLE
"Else" clause for Continue If action

### DISCUSSION THREAD

**Discussion by micebrain** [Budibase/budibase#8990] [Ideas]
Hello,
I find the "Continue If/Stop If" action really useful especially when checking API response.
What I think could be useful is to add an optional `else` clause for "Continue If" to show notification or an option to show notification for "Stop If". Currently the "Show Notification" is a separate action and it makes it difficult to use together with the "Continue If/Stop If" action.

Something like
![image](https://user-images.githubusercontent.com/7136209/206709682-70b11550-1671-4299-ae32-b2eb16d2b344.png)

My use case is to check the response of an API and continue if success and show a notification if error is encountered and stop the rest of the action. 

Thanks!

**Comment by melohagan** [comment]
It certainly would be nice! There's a similar issue here: https://github.com/Budibase/budibase/issues/7016
